### ISSUE TITLE
Followers/Following NaN in store view tabs

### ISSUE THREAD

**Issue by placer14** [OpenBazaar/openbazaar-desktop#1553]
Brief Description: Followers/Following appears as NaN on the store view

Operating System (OS and version): MacOS
OpenBazaar version (shown on `About OpenBazaar` page in menu): 
Hardware: 2.2.5/0.12.4 (rc1)

Reproducible (Always / Almost Always / Sometimes / Rarely / Couldn't Reproduce):  Always

Steps to reproduce:
1. Load ob://QmVFKJ946LuFdx2bqLCxhQfEbCqcGwkn2UEk32qXEESBrD/home

Observed Behavior:
<img width="697" alt="screenshot 2018-09-06 12 06 19" src="https://user-images.githubusercontent.com/305332/45170404-d1bc7100-b1cd-11e8-9fc7-0791bd31840a.png">

Expected Behavior:

Additional info (links, images, etc go here):

**Comment by placer14** [comment]
Request to server API `http://localhost:4002/ob/profile/QmYM4A7pC8L8NJEKCHDjxRssnw3XzM2LfaXQPmGPEXoWWr` returns
```
{
    "peerID": "QmYM4A7pC8L8NJEKCHDjxRssnw3XzM2LfaXQPmGPEXoWWr",
    "handle": "",
    "name": "MG (OB1)",
    "location": "",
    "about": "",
    "shortDescription": "This account is managed by MG (OB1).",
    "nsfw": false,
    "vendor": true,
    "moderator": false,
    "contactInfo": {
        "website": "",
        "email": "",
        "phoneNumber": "",
        "social": [
        ]
    },
    "colors": {
        "primary": "#FFFFFF",
        "secondary": "#ECEEF2",
        "text": "#252525",
        "highlight": "#2BAD23",
        "highlightText": "#252525"
    },
    "avatarHashes": {
        "tiny": "zb2rhik8Q8k7aa4dME6FqKTyeUpZW3Eg7UFMKpjJvxNSzNref",
        "small": "zb2rhhk7TcTjqLBNbWWaTjjjwHCRxQMiBQiJGPBUax4G2DKgp",
        "medium": "zb2rhbEEShD6dXasHUwyJJaydbbav5qnfrw7FPLz5cCkkgsuG",
        "large": "zb2rhdbzMttSTcxeY5MMBRw1E7jVvP8f7TTjLkHvr1znJk3kM",
        "original": "zb2rhagaBhr7yKNDpou451DntoQgAz4vXkEkzrAhQUFXcXKJg"
    },
    "stats": {
        "followerCount": 1,
        "followingCount": 1,
        "listingCount": 1,
        "ratingCount": 0,
        "postCount": 0,
        "averageRating": 0
    },
    "bitcoinPubkey": "03aaf9e19cc3e0dea1ba3ee92968bdf1cf28a41a6a2e0cc1a0a2e9e3037eeaeef6",
    "lastModified": "2018-09-06T12:22:12.427693506Z",
    "currencies": [
        "BCH"
    ]
}
```

**Comment by jjeffryes** [comment]
Closed in #1617
